anaconda.py

```diff
@@ -80,13 +80,12 @@ def exitHandler(rebootData, storage):
     if anaconda.payload:
         anaconda.payload.unsetup()
 
-    # Unmount the filesystems.
-    if not conf.target.is_hardware:
-        anaconda.storage.umount_filesystems(swapoff=False)
+    from pyanaconda.modules.common.task import sync_run_task
+    storage_proxy = STORAGE.get_proxy()
 
-    # Tear down disk images.
-    if conf.target.is_image:
-        anaconda.storage.devicetree.teardown_disk_images()
+    for task_path in storage_proxy.TeardownWithTasks():
+        task_proxy = STORAGE.get_proxy(task_path)
+        sync_run_task(task_proxy)
```

Comment:
I think this could be as a function like ``sync_run_all_tasks()``.

Reply:
The teardown should be eventually handled by the boss, so I am not going to implement the new function for now.

Reply:
OK, I still think it has it's usages on the other places but that would be better done in a separate PR.